File: back/Authentication/Entities/CompanyAuth.cs
namespace Authentication.Entities;

public class CompanyAuth
{
    public required int Id { get; set; }
    public required string Name { get; set; }
    public int BusinessId { get; set; }
    public Business? Business { get; set; }
    public ICollection<CompanyUserAccount> CompanyUserAccounts { get; set; } = new List<CompanyUserAccount>();
}

File: front/src/components/company/dtos/company-auth.ts
import {CompanyUserAccount} from '../../authentication/dtos/company-user-account';

export class CompanyAuth {
  id!: number;
  businessId!: number;
  name!: string;
  tradeName!: string;
  // companyProfileId!: string;
  cnpj!:string;
  companyUserAccounts!: CompanyUserAccount[];
  deleted!: Date;
  Registered!: Date;

}
